extras/check_code_style.sh

```diff
@@ -232,7 +239,13 @@ for f in $file_names; do
     *.py )
       echo "Check PEP8 on file $f:"
 
-      if ! pep8_result=`pep8 --first --ignore=$PEP8_IGNORES $f` ; then
+      if [[ $f =~ $EXAMPLE_DIRS ]]; then
```

Comment:
@apeyser As far as I have been able to find out, the only possibility is to append `# noqa` to pertaining lines. I have used that very sparingly. I have not been able to find a way to disable specific PEP8 errors for individual files or sections of files by comments in the files. In example code that will go onto our website or scripts that are included in the Topology manual, the `# noqa` comment seems undesirable because it would distract readers.